src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,22 @@
+package wolox.training.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class BookController {
```

Comment:
No need to change it because in the future you'll be adding Book functionalities, but keep in mind that the name of the controller should always be descriptive of the functionalities it contains (in this case, the controller could be name for example GreetingController) 😄